integration-tests/e2e/kubetest/description/1.21/skip.json

```diff
@@ -675,5 +675,11 @@
   { "testcase": "[sig-instrumentation] MetricsGrabber should grab all metrics from API server.", "description": "Depends on control plane (kapi,kcm,...) being present in 'kube-system' ns of the target cluster (shoot), fails since 1.20 if cp pods are not there: https://github.com/kubernetes/kubernetes/commit/20863a8b4390d275925338571e7a46105177a6cf" },
   { "testcase": "[sig-instrumentation] MetricsGrabber should grab all metrics from a ControllerManager.", "description": "Depends on control plane (kapi,kcm,...) being present in 'kube-system' ns of the target cluster (shoot), fails since 1.20 if cp pods are not there: https://github.com/kubernetes/kubernetes/commit/20863a8b4390d275925338571e7a46105177a6cf" },
   { "testcase": "[sig-instrumentation] MetricsGrabber should grab all metrics from a Kubelet.", "description": "Depends on control plane (kapi,kcm,...) being present in 'kube-system' ns of the target cluster (shoot), fails since 1.20 if cp pods are not there: https://github.com/kubernetes/kubernetes/commit/20863a8b4390d275925338571e7a46105177a6cf" },
-  { "testcase": "[sig-instrumentation] MetricsGrabber should grab all metrics from a Scheduler.", "description": "Depends on control plane (kapi,kcm,...) being present in 'kube-system' ns of the target cluster (shoot), fails since 1.20 if cp pods are not there: https://github.com/kubernetes/kubernetes/commit/20863a8b4390d275925338571e7a46105177a6cf" }
+  { "testcase": "[sig-instrumentation] MetricsGrabber should grab all metrics from a Scheduler.", "description": "Depends on control plane (kapi,kcm,...) being present in 'kube-system' ns of the target cluster (shoot), fails since 1.20 if cp pods are not there: https://github.com/kubernetes/kubernetes/commit/20863a8b4390d275925338571e7a46105177a6cf" },
+  { "testcase": "[sig-node] RuntimeClass should support RuntimeClasses API operations [Conformance]"},
```

Comment:
a conformance test MUST never be skipped.
otherwise we loose k8s conformance certification for the gardener clusters

Reply:
thanks, i've fix this

Reply:
Hi @schrodit , i have fixed the changes you mentioned, could you please re-review this PR? if all good please merge this one, i don't have write access to this repo
thanks